### ISSUE TITLE
AoTTG2 Titan Eating/Killing Animations

### ISSUE THREAD

**Issue by Prolonpo** [AoTTG-2/AoTTG-2#310]
**How will this enhancement improve the game?**
Idea from @OrangeMeep_ An improvement to the Titan eating and killing animations as those animations will be dependent on the height of the titan. Will introduce variability of the eating animations and the corpses left behind.

**Describe the solution you'd like**
Please see the attached file:
[15 OrangeMeep__AoTTG2_Titan_Kill_animation_ideas..pdf](https://github.com/AoTTG-2/AoTTG-2/files/5396189/15.OrangeMeep__AoTTG2_Titan_Kill_animation_ideas.pdf)

**Describe alternatives you've considered**
N/A

**Contributor instructions** (Do not modify)
1. Determine if the enhancement is compatible with existing features
2. If the idea is denied, please provide a reason and close the issue. If the idea is accepted, consult your team to do an estimation of effort. Remove the [PROPOSAL] name and label.
3. Move the issue to the Future Milestone. On rare exceptions an issue may be moved to Release if there's sufficient upvotes.
4. Once the issue matches the current milestone, follow development procedures.

**Comment by Prolonpo** [comment]
@bLinK:
• Titan's stomps and footsteps can kill human players
• Titan will be focused on eating instead of attacking
• As for the smashing move of the normal titan, I'll also include it, but leave it up for debate between staff members

Reference post: https://discord.com/channels/681641241125060652/683376807101071489/778270179502194688

@EEE:
• Falling titans will be able to crush and kill human players - @Dakros added that ragdoll physics should apply to killed titans

Reference post: https://discord.com/channels/681641241125060652/683376807101071489/778715203818094623

Summary post: https://discord.com/channels/681641241125060652/683107211983323164/780830291089686528

**Comment by Prolonpo** [comment]
From @AlexBot and related to #326 

Death Sounds 

In the current state of the AoTTG there are no sounds like grunts or screams upon death of Player or Titan. In the game there is "splash" which is played upon death of Player and it's okay when we get slapped to death my titan or the moment we are getting eaten but other wise our character displays no signs of being afraid of injured. Also if we do anything to the titans they just simply don't care, most they do is silently yawn

But while being grabbed (Player) or injuring a Titan in some way (Blinding them, or even killing them) a sound would play from a set of Default one (3-5) that would be randomly decided

Also few limitations for those sounds:
-Length of the file would be only around 2 seconds
-There would be a limited range
-There can be only a certain amount of those sounds (from 3 to 5 but that number can change)
-There would be also possibility for a sound to not play at all

Having those sounds would add a lot to sounds the almost silent game we have for now but having too much of them would create unwanted noise. 

This whole video can be a great example. Soldiers scream while being grabbed. Titans grunt after being killed and they scream after being blinded 
https://www.youtube.com/watch?v=fHvqaBQhZhw

In short 
Sounds for situations like getting grabbed by a titan, injuring a Titan or killing it (With few limitations)

Reference post: https://discord.com/channels/681641241125060652/683376785659789342/790154496306184192

**Comment by Prolonpo** [comment]
From @Mujib, here are some additional animations for titan's when they are stuck (which involves reaching towards the player and trying to inch through a narrow opening), drifting, as well as hiding behind cover to observe passing players.

Reference post (start): https://discord.com/channels/681641241125060652/681647375336341522/782284314964131890

Reference post (stop): https://discord.com/channels/681641241125060652/681647375336341522/782285743552593930

**Comment by Prolonpo** [comment]
From @Theodor, titans falling animation after being killed should be dependent on what stage of an animation they're in when they get struck.

Prolonpo's note: For example, if a titan is currently doing a slap, killing this titan during its slap animation should carry on the slap momentum instead of playing the default animation.

https://discord.com/channels/681641241125060652/683376807101071489/787731735899013131

**Comment by Prolonpo** [comment]
From @Stiffy, aberrants should be able to perform a horizontal dolphin dive to catch players.

https://discord.com/channels/681641241125060652/683376807101071489/797912383736512595

**Comment by Prolonpo** [comment]
Additional ideas were brought to this issue and the first proposal meeting didn't bring any conclusive decision on this matter, request for the Proposal Team to investigate on this issue
- Might want to reach out to the Art Team in order to see which elements have been completed so far

1st Proposal Meeting's notes:
![image](https://user-images.githubusercontent.com/72053473/127580868-4cf03e37-a8bc-43a2-99cf-d236b2e59d8e.png)
